### PR TITLE
Replace XmlTextWriter as it's not supported in NETCore

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Common/XHtmlWriter.cs [hunk 1]
```diff
@@ -7,14 +7,17 @@ namespace Microsoft.DocAsCode.Common
     using System.IO;
     using System.Xml;
 
-    public class XHtmlWriter : XmlTextWriter
+    public class XHtmlWriter : XmlWriter
     {
         private readonly HashSet<string> _voidElements;
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Common/XHtmlWriter.cs]: >_voidElements [](start = 41, length = 13)

could be static?

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by vwxyzh** [comment]
:+1:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**